test/registries.conf

```diff
@@ -21,3 +21,7 @@ location="quay.io/libpod"
 [[registry]]
 location="localhost:5000"
 insecure=true
+
+# Alias used in tests.
+[aliases]
+  "podman-desktop-test123"="florent.fr/will/like"
```

Comment:
is the indentation desired?

Reply:
That's how the `000-shortnames.conf` file is formatted, so I just copied it over.